### DISCUSSION TITLE
Error Message After Update

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#1645] [Fixed Issues]
@rern
I got this message following a successful 20240330 update:

/srv/http/bash/status.sh: line 495: ((: < 1000000 : syntax error: operand expected (error token is "< 1000000 ")
awk: cmd. line:1: BEGIN { printf "%.2f", /1000000 }
awk: cmd. line:1:                         ^ unterminated regexp
awk: cmd. line:1: BEGIN { printf "%.1f", /1000 }
awk: cmd. line:1:                         ^ unterminated regexp

**Comment by findnix** [comment]
... same here on one of three instances.
Do you need some traces?

**Comment by Roland-B** [comment]
@rern  Error still present in latest update!?

/srv/http/bash/status.sh: line 496: ((: < 1000000 : syntax error: operand expected (error token is "< 1000000 ")
awk: cmd. line:1: BEGIN { printf "%.2f", /1000000 }
awk: cmd. line:1:                         ^ unterminated regexp
awk: cmd. line:1: BEGIN { printf "%.1f", /1000 }
awk: cmd. line:1:           

Roland.              

rAudio 20240408
6.6.18-1-rpi aarch64
Raspberry Pi 4B Rev 1.2
BCM2711 • 3.7 GiB
4 x Cortex-A72 @ 1.50 GHz

**Comment by R101G** [comment]
@rern Same here. It did not happen with the previous update.
